runtime/doc/options.txt

```diff
@@ -5560,7 +5560,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 	old Vi scripts.
 
 						*'renderoptions'* *'rop'*
-'renderoptions' 'rop'	Removed. {Nvim} will choose the best renderer available.
+'renderoptions' 'rop'	Removed. {Nvim}
```

Comment:
I think adding reasoning for removal is a good idea, but not here. This is the only removed option with a description, and it's incomplete. Perhaps we can just replace `vi_diff.txt` with `nvim_diff.txt`, then tack on `(See |nvim-diff-options|)` to the end of each removed option in `options.txt`.

I can convert this PR, or just open a new one for a cleaner comment history.